### DISCUSSION TITLE
Possible Upgrades?

### DISCUSSION THREAD

**Discussion by christianaz** [FrandsenGroup/beams#212] [Ideas]
So I'm not going to be using BEAMS as much anymore, but I thought I'd give some possible features to add in case you guys get bored. :)

1) Rename the files. When you download the data using BEAMS, all of the files are labeled with the run number (they are labeled with the run number on the computer, the files in BEAMS have the run information as well). This isn't really helpful information when I want to chose certain runs off my computer to fit, since the run numbers are too long to easily memorize. What I have been doing is going through one by one and renaming each run with the type of run and the temperature, but it would be handy if there was some way to automatically change the file names so I could actually see the run information rather than just the number. Let me know if this makes sense.
2) The window sizing is still having some problems, but I'm not sure how y'all would be able to fix that. If I try to snap the window to fill half the screen, it fills about 3/4, which doesn't work well if I want to have another window open. It could just be something interesting to fiddle with.
3) Make it so you can copy the chi-squared value. For some reason, I can copy all of the other values in the table, but the chi-squared one won't let me touch it. It just means that I have to copy it by hand, but it could be handy to be able to copy it.
4) I still have never seen the histogram feature work, but I see that the issue has a bounty on it now, so I assume it'll get fixed eventually :)

Thanks so much for all the work you've put into this program!

**Comment by aPeter1** [comment]
Thanks for all your testing and input!
